lib/routes/sms.js

```diff
@@ -106,8 +106,19 @@ module.exports = (log, isA, error, config, customs, sms) => {
           .then(reply, reply)
 
         function getLocation () {
+          if (request.app.clientAddress === '127.0.0.1') {
```

Comment:
How about something like: what if we have a config switch for the `getLocation()` call, and we disable the check in local dev config.